driving-game/main.js

```diff
@@ -70,9 +73,16 @@ document.body.addEventListener('keydown', function (event) {
   }
 })
 
+let isRunning = false
```

Comment:
Rather than a global variable, make the car responsible for keeping track of this.